packages/webpack/src/buildtime/types.d.ts

```diff
@@ -0,0 +1,28 @@
+import type { LavaMoatPolicy } from 'lavamoat-core'
+import type { LockdownOptions } from 'ses'
+
+export interface ScuttlerObjectConfig {
+  enabled?: boolean
+  exceptions?: string[]
+  scuttlerName?: string
+}
+
+export type ScuttlerConfig = ScuttlerObjectConfig | boolean | undefined
+
+export interface LavaMoatPluginOptions {
```

Comment:
each member here should have a description, but it doesn't have to happen in this PR